src/der.rs

```diff
@@ -277,4 +317,11 @@ mod tests {
             });
         }
     }
+
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha256_key_encoded_ber,
+                               "rsa-pkcs1-sha256-key-encoded-ber.pem" );
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha1_bad_key_der_length,
+                               "rsa-pkcs1-sha1-bad-key-der-length.pem");
+    test_parse_bad_spki_der!(test_rsa_pkcs1_sha1_bad_key_der_null,
+                               "rsa-pkcs1-sha1-bad-key-der-null.pem");
```

Comment:
These are failing on android because the files are not present. I'll see if I can track down why.